### DISCUSSION TITLE
What settings do you use in UVR 5 to remove extraneous sounds? (Ultimate Vocal Remover)  I need to leave only the voice in the audio to create an RVC (voice model)

### DISCUSSION THREAD

**Discussion by kotkakstus** [Anjok07/ultimatevocalremovergui#1161] [Q&A]
What settings do you use in UVR 5? I didn’t really understand it, so I only showed it on video, or experimented quite a bit. In general, tell those who worked in UVR 5 what settings can be set to remove all extraneous sounds from the voice such as creaks, blows and other sound effects. (I have never found complete, NON-BASIC guides on them). I need to leave only the voice in the audio to create an RVC (voice model), perhaps there are some errors in the text or inaccuracy since I am writing this through Google translator.

**Comment by squidboi666** [comment]
I've been making making models using audio from streamers, and the only consistent way I've been able to remove 100% of the gameplay audio is by using these settings:

Process Method & Model: MDX-Net Kim Vocal 1 
Segment Size: 2240
Overlap: 0.99

I'm also enabling GPU Conversion & Vocals Only.


It will take a LONG time to process, but this is how I've been getting near perfect output. The most important setting seems to be the Overlap setting.

**Reply by kotkakstus** [reply]
Thanks, I'll try.

**Reply by kotkakstus** [reply]
Can you please try to do this on your PC? (mine didn’t work)                                                                                                                                               
 1)This is the first voice https://drive.google.com/file/d/1zlj7cJO60FHHaK3y90Q0DIC4z0N8RAls/view?usp=sharing                                                               
    2)This is the second voice https://drive.google.com/file/d/1EU5PLvocs8yLLz5nlE9L8n9Rv2uJvMnn/view?usp=sharing                                                  
 This is again the first one, but already cleared of music, maybe this way the unnecessary sounds will be removed better                                                           https://drive.google.com/file/d/1duUM_erawu0no7IVr28qp-ipWKrAJbMx/view?usp=sharing                                                                                            I will be very grateful if you help me, I have been suffering for 2 months now.

**Comment by GUUser91** [comment]
I found a music audio speech separator application that is good at removing or at least massively reducing the sound effects of spoken audio. I'm using the BandIt Plus model and config_dnr_bandit_bsrnn_multi_mus64.yaml file. I'm also using the audacity plugin, Acon Digital DeVerberate 3, to reduce the reverb. Note for the second input audio example, the louding screaming speech was in the effects output file for some reason.
https://github.com/ZFTurbo/Music-Source-Separation-Training/
Here's a sample.
Input audio.
https://vocaroo.com/1kW5ouqlmNEJ
https://vocaroo.com/1jdwZKVDNTpd
Output audio
https://vocaroo.com/1e4Hdj50P1py
https://vocaroo.com/19dlBzlnb5sv

Edit: I just discovered the moises.ai application. I purchase the yearly pro plan since it was on sale for $150. This deal last until Decemember 8. It does a much better job at separating sound effects from spoken / speech audio.
Input audio
https://vocaroo.com/1ipKWeiyr3o6
https://vocaroo.com/1jdwZKVDNTpd
Output audio
https://vocaroo.com/1hkFYE87glzr
https://vocaroo.com/1fEm7FwtIQrz

Edit I found a free alternative that lets you use sam audio with less vram requirements.
https://github.com/0x0funky/audioghost-ai
